content/tutorials/token-authentication.textile

```diff
@@ -422,17 +422,17 @@ blang[swift].
 
 h2. Step 4 - Respond to authentication requests
 
-Before we move onto the client, you'll need to first make sure the web server is capable of issuing token requests to clients that need to authenticate with Ably. However, instead of jumping into code, let's quickly talk about how token authentication works and what the difference is between a "@Token@":https://ably.com/documentation/realtime/authentication#token-details and "@TokenRequest@":https://ably.com/documentation/realtime/authentication#token-request is.
+Before we move onto the client, you'll need to first make sure the web server is capable of issuing token requests to clients that need to authenticate with Ably. However, instead of jumping into code, let's quickly talk about how token authentication works and what the difference is between a "@Token@":https://ably.com/docs/realtime/authentication#token-details and "@TokenRequest@":https://ably.com/docs/realtime/authentication#token-request is.
```

Comment:
`/docs/api/realtime-sdk/authentication#token-details` and `/docs/api/realtime-sdk/authentication#token-request`